code/src/main/java/com/onlinecontacttracing/messaging/filters/CustomizeMessageTriesFlaggingFilter.java

```diff
@@ -0,0 +1,21 @@
+package com.onlinecontacttracing.messaging.filters;
+
+import java.util.ArrayList;
+import com.onlinecontacttracing.messaging.filters.FlaggingFilter;
+import com.onlinecontacttracing.storage.PositiveUser;
+import com.onlinecontacttracing.storage.PotentialContact;
+import java.lang.Exception;
+
+//Checks if user has attempted to submit a message too many times
+public class CustomizeMessageTriesFlaggingFilter implements FlaggingFilter{
+  private int maxNumberOfTries = 3; 
```

Comment:
for a hard coded value like this, it should be a private static final in and should be named MAX_NUMBER_OF_TRIES

Reply:
gotcha